Include/internal/pycore_intrinsics.h

```diff
@@ -1,26 +1,24 @@
+// Auto-generated by Tools/build/generate_opcode_h.py from Lib/opcode.py
 
 /* Unary Functions: */
+#define INTRINSIC_INVALID                        0
+#define INTRINSIC_PRINT                          1
+#define INTRINSIC_IMPORT_STAR                    2
+#define INTRINSIC_STOPITERATION_ERROR            3
+#define INTRINSIC_ASYNC_GEN_WRAP                 4
+#define INTRINSIC_UNARY_POSITIVE                 5
+#define INTRINSIC_LIST_TO_TUPLE                  6
 
-#define INTRINSIC_PRINT 1
-#define INTRINSIC_IMPORT_STAR 2
-#define INTRINSIC_STOPITERATION_ERROR 3
-#define INTRINSIC_ASYNC_GEN_WRAP 4
-#define INTRINSIC_UNARY_POSITIVE 5
-#define INTRINSIC_LIST_TO_TUPLE 6
-
-#define MAX_INTRINSIC_1 6
+#define MAX_INTRINSIC_1                          6
 
 
 /* Binary Functions: */
+#define INTRINSIC_INVALID                        0
```

Comment:
Looks like INTRINSIC_INVALID is defined twice?

Reply:
Updated to differentiate between invalid for the two `CALL_INTRINSIC_*` opcodes.